docs/excel/excel-add-ins-comments.md

```diff
@@ -197,8 +197,121 @@ Excel.run(function (context) {
 });
 ```
 
+## Comment events
+
+Your add-in can listen for comment additions, changes, and deletions. [Comment events](/javascript/api/excel/excel.commentcollection#event-details) occur on the `CommentCollection` object. To listen for comment events, register the `onAdded`, `onChanged`, or `onDeleted` comment event handler. When a comment event is detected, use this event handler to retrieve data about the added, changed, or deleted comment. The `onChanged` event also handles comment reply additions, changes, and deletions. 
```

Comment:
Can we add a line to this overarching section about multiple comments? Something like "Each comment event only triggers once when multiple additions, changes, or deletions are performed at the same time. All the `Comment*EventArgs` objects contain arrays of comment IDs to map the event actions back to the comment collections."